vscode/src/status.ts

```diff
@@ -17,10 +22,11 @@ export abstract class StatusItem {
   public item: vscode.LanguageStatusItem;
 
   constructor(id: string) {
-    this.item = vscode.languages.createLanguageStatusItem(id, {
-      scheme: "file",
-      language: "ruby",
-    });
+    const documentSelector = SUPPORTED_LANGUAGE_IDS.map((language) => ({
```

Comment:
Should this be plural? `documentSelectors`?

Reply:
I matched the type name, which is `DocumentSelector` and is an array of entries.